index.htlm

```diff
@@ -11,7 +11,7 @@
 <body>
   <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-  <nav id="navbar">
+  <nav id="navbar" class="navbar">
```

Comment:
- [ ] Great job asking politely for the change! 💯 However, a good review should also contain an explanation of "why" you think is a good idea to implement this change. Please consider editing this comment to include this explanation! 🤓 🔧 

>This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
>
>This request lacks a "why": Why don't you use an article tag here instead of a div?